sections/components.xml

```diff
@@ -200,22 +200,52 @@ in the section entitled "GNU Free Documentation License".
     component. The differences are a few minor, but salient, points.</para>
 
     <para><link linkend="com-namespaces"><emphasis
-    role="bold">Namespaces</emphasis></link>. All native Joomla 4 component
+    role="bold">Namespaces</emphasis></link>. All native Joomla 4 components
     use <link
     xlink:href="https://www.php.net/manual/en/language.namespaces.php">PHP
-    namespaces</link> for their PHP code. This is a very simple change in
-    thinking for models, views and controllers. For example, instead of your
+    namespaces</link> for their PHP code. This is a simple change in naming
+    classes for models, views and controllers. For example, instead of your
     frontend model being called <code>ExampleModelItem</code> it will now be
     the class <code>ItemModel</code> in the namespace
-    <classname>\<replaceable>My</replaceable>\<replaceable>Prefix</replaceable>\Site\Model</classname>
-    where <code>\My\Prefix</code> is something you can define, and has no
-    restrictions other than it needs to be a valid PHP namespace, unique to
-    your component and should not start with <code>\Joomla</code> (component
-    namespaces starting withIn <code>\Joomla</code> are reserved for core
-    components). This also applies to custom form fields you may need to
-    create in your component — and there <emphasis>is</emphasis> a way to tell
-    Joomla which namespace to look custom form fields in with new attributes
-    in XML forms.</para>
+    <classname>\<replaceable>MyComponentNamespace</replaceable>\Site\Model</classname>
```

Comment:
Strange the .xml~ backup file had come through, as it was in the .gitignore. Sorry, hadn't seen it.

I'll change the MyComponentNamespace to what you call a more realistic example, BUT in this comment and subsequent ones you still made a mistake with the right order: it is NOT \Acme\Example\Component, but Acme\Component\Example, so NOT \Component after the \Package name, but BEFORE it!

I used "<vendor name>" instead of just "vendor" to avoid confusion with literally using the word "vendor" (as is for instance used as a directory in /libraries), but maybe the angle brackets can be confusing too... will play a bit with it, using as much as possible a concrete example like \Acme\Component\Example.

Reply:
Yes, sorry, I was typing really fast because I had to leave soon to go get my kid from the school bus :)

Regarding brackets, it's not needed because we use the `<repleceable>` tag. If it's not clear I can add the brackets and/or format it in a different way with CSS (the conversion to HTML wraps it in a span with a different class). That's why I am trying to use the DocBook 5 XML features as much as possible; they may slow me down a little bit when writing the documentation but they make improving the output infinitely easier with bog standard CSS.